base/pkg/dir.jl

```diff
@@ -26,21 +26,26 @@ function cd(f::Function, args...; kws...)
     metadata_dir = joinpath(dir, "METADATA")
     if !isdir(metadata_dir)
         !haskey(ENV,"JULIA_PKGDIR") ? init() :
-            error("Package metadata directory $metadata_dir doesn't exist; run Pkg.init() to initialize it.")
+            throw(PkgError("Package metadata directory $metadata_dir doesn't exist; run Pkg.init() to initialize it."))
+    end
+    try
+        Base.cd(()->f(args...; kws...), dir)
+    catch err
+        if isa(err, PkgError)
+            print_with_color(:red, "PkgMgr Error: $(err.msg)")
```

Comment:
It was my initial version, but `Package manager error` was too long considering some error messages. I'll make it just 'ERROR`